mcs/class/Mono.Profiler.Log/Mono.Profiler.Log/LogEnums.cs

```diff
@@ -199,4 +201,22 @@ public enum LogSynchronizationPoint {
 		WorldStop = 1,
 		WorldStart = 2,
 	}
+
+	public enum HeapRootKind {
+		External = 0,
+		Stack,
+		FinalizerQueue,
```

Comment:
I wonder if it matters, the distinction today exists as each get a separate pseudo-address when their roots get registered.

Given that the vast majority of C# users don't know that critical finalizers exists, I think it's ok to keep them all on the sabe basket until it turns to be an issue.